beer.rb

```diff
@@ -1,11 +1,22 @@
 require 'pry'
- 
+
 #write a function that accepts a variable: age
 #if age is greater than 21, display "Yay, have a beer!"
 #if age is less than 21, display "Nay, don't have a beer!"
- 
+
+def drinking_age(age)
+  if age.to_i > 21
+    print "Yay, have a beer!"
+  elsif age.to_i < 21
```

Comment:
Hey @misswysocki you could have used else here instead of elsif with a condition, since we're only trying to check if they're older than 20 or equal to or older than 21.

Consider the following:

```ruby 

if age.to_i >= 21
  print "yay have a beer!"
else 
 print "nay don't have a beer"
end 
```